### PR TITLE
KAA-1616: [Reviewed] Using Kaa C++ SDK, Edison. [master]

### PR DESCRIPTION
**Pull request by hunch7**

### CODE REVIEW

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md [hunk 1]
```diff
@@ -9,196 +9,196 @@ sort_idx: 60
...
+{:.note}
 
-If this is the first time you use the Intel Edison technology, you have to start with configuring your board.
-For this purpose, refer to the official Edison getting started guide.
-After finishing with the guide, you will be able to access your board through both a serial terminal and the ssh client.
+## Prerequisites
 
->**Note:** The instructions below are for the official Intel Edison firmware, Poky (Yocto Project Reference Distro) 1.7.3, kernel version 3.10.17-poky-edison+.
+To install all required third-party components, perform the instructions below directly on the Edison board.
+Some steps requre root permissions.
```
Comment (Polinaa) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md]: require

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md [hunk 2]
```diff
@@ -9,196 +9,196 @@ sort_idx: 60
...
+   ${HOST_PYTHON} -E configure.py --cpu=x86_32 --cc-bin=${CROSS_COMPILE}g++ --prefix=${SDKTARGETSYSROOT}/usr
+   make && make install
+   ```
+
     Copy Botan object files `${SDKTARGETSYSROOT}/usr/lib/libbotan*` to `/usr/lib` directory of Edison board.
 
-1. Download C++ SDK from your [Kaa instance]({{root_url}}Glossary/#kaa-instance-kaa-deployment).
-1. Compile C++ SDK.
+8. Download Kaa C++ SDK from your [Kaa instance]({{root_url}}Glossary/#kaa-instance-kaa-deployment) and compile it.
+See [Generate SDK](Programming-guide/Your-first-Kaa-application/#generate-sdk).
```
Comment (Polinaa) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md]: Page not found

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md [hunk 3]
```diff
@@ -9,196 +9,196 @@ sort_idx: 60
...
-        cd build
-        cmake -DCMAKE_TOOLCHAIN_FILE=../toolchains/edison.cmake ..
-        make
-        
-Now you are ready to cross compile your own Kaa application but don't forget to use proper toolchain.
+## Build Kaa application
+
+After you installed the required dependencies and built the C++ SDK, you can build and run your [Kaa application]({{root_url}}Glossary/#kaa-application).
+
+Since Edison runs on Linux, you can use the [Linux guide]({{root_url}}Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Linux/#build-c-sdk) to build and run your application.
```
Comment (Polinaa) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md]: #build-kaa-application in C++ Linux guilde

**Review by Polinaa — APPROVED** [review]

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md (hunk 1 above)
Comment (hunch7) [reply doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md]: Fixed.

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md (hunk 2 above)
Comment (hunch7) [reply doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md]: Fixed.

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md (hunk 3 above)
Comment (hunch7) [reply doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md]: Fixed.

**Review by vadimol — APPROVED** [review]

**Pull request merged**